eg/Classes/PluginInstall.py

```diff
@@ -131,6 +206,28 @@ def GetPluginData(self, pluginInfo):
             "icon": iconData,
         }
 
+    def BackupPlugin(self, pluginInfo):
+        backupName = (
+            '%s-%s.egplugin' % (pluginInfo.englishName, pluginInfo.version)
+        )
+
+        backupPath = os.path.join(
+            eg.folderPath.ProgramData,
+            'EventGhost',
+            'PluginBackups'
+        )
+        if not os.path.exists(backupPath):
+            os.makedirs(backupPath)
+
+        eg.PrintNotice('Creating backup of plugin.')
+        eg.PrintNotice('Writing ' + os.path.join(backupPath, backupName))
```

Comment:
I would prefer to see an info icon in the log instead of the yellow triangle. It looks to me like a warning that something could be wrong. (eg.Print instead of eg.PrintNotice). Just my opinion.